docker/test/integration/minifi/core/DockerTestCluster.py

```diff
@@ -54,13 +55,16 @@ def get_app_log(self, container_name):
 
         return container.status, None
 
-    def wait_for_app_logs(self, container_name, log_entry, timeout_seconds, count=1):
+    def wait_for_app_logs(self, container_name, log_entry, timeout_seconds, count=1, use_regex=False):
         wait_start_time = time.perf_counter()
         while (time.perf_counter() - wait_start_time) < timeout_seconds:
             logging.info('Waiting for app-logs `%s` in container `%s`', log_entry, container_name)
             status, logs = self.get_app_log(container_name)
-            if logs is not None and count <= logs.decode("utf-8").count(log_entry):
-                return True
+            if logs is not None:
+                if not use_regex and count <= logs.decode("utf-8").count(log_entry):
+                    return True
+                elif use_regex and len(re.findall(log_entry, logs.decode("utf-8"))) <= count:
```

Comment:
this should be `>=`

(I would also change the non-regex case from `count <= something` to `something >= count` as I think that would read better)